extensions/amp-consent/0.1/tcf-api-command-manager.js

```diff
@@ -102,14 +184,14 @@ export class TcfApiCommandManager {
    * @param {!Window} win
    * @param {!JsonObject} returnValue
    * @param {string} callId
-   * @param {boolean=} success
+   * @param {boolean=} opt_success
```

Comment:
nit: can you change this to use a default value instead?

Reply:
`success` can be `undefined`, `true`, or `false`, so I think it's cleaner to have it be optional, rather than passing in `undefined` or explicitly setting it `sendTcfApiReturn_(..., sucess = undefined)`.